core/src/style/pointStyle.h

```diff
@@ -50,6 +53,8 @@ class PointStyle : public Style {
 
     UniformLocation m_uTex{"u_tex"};
     UniformLocation m_uOrtho{"u_ortho"};
+
+    mutable std::unique_ptr<LabelMesh> m_mesh;
```

Comment:
What about `DynamicLabelMesh`, to make it clear that the content is updated every frame.